src/app/components/request-password.tsx

```diff
@@ -14,6 +15,15 @@ import { Page } from '@app/ui/layout/page/page.layout';
 
 import { ErrorLabel } from './error-label';
 
+const waitingMessages: WaitingMessages = {
```

Comment:
Thanks for re-adding this @fbwoolf . I removed it specifically as we had no design for the warning text and it looked really incorrect - huge font. 

Better to keep it if we need it and style it properly